### ISSUE TITLE
Deleting a project doesn't reset the right hand side so you can still create New Tasks which results in an error

### ISSUE THREAD

**Issue by graydwarf** [Mad-Cookies-Studio/mad-productivity#13]
Repro Steps:
- Click Tasks
- Click New (so new project created)
- Click New Project (so task list shown at right)
- Click Delete button (X) next to project
- Click New Task button at right

Expected:
- Page at right should prolly be hidden once project is deleted.

Actual:
- Page remains interactable and leaves user in wonky state where unhandled errors occur

Error:
Error: Attempt to call function 'set_percent_done' in base 'null instance' on a null instance.

File:
ToDo.gd

Line:
```
 update_percent_done(p_id : int, child_p_id : int) -> void:
$VBoxContainer/HSplitContainer/PanelL/ScrollContainer/ProjectButtons.get_child(child_p_id).set_percent_done(res.get_percent_done(p_id))

```

**Comment by graydwarf** [comment]
Should note, not a blocker for me. Just poking around.

**Comment by lentsius-bark** [comment]
This is well spotted. The right panel should reset it's views and settings so that the aforementioned does not happen! Thanks for reporting.

**Comment by lentsius-bark** [comment]
fixed by c5ee39407ea5078ac7f3cd15496d47cb486c6d2e
